mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/project.py

```diff
@@ -1,17 +1,14 @@
-import json
 import os
-import urllib.request
+from abc import abstractmethod
 from dataclasses import dataclass
 from typing import Dict, List
 
 from osgeo import ogr
 
 from mapswipe_workers.definitions import DATA_PATH, CustomError, logger
-from mapswipe_workers.firebase.firebase import Firebase
 from mapswipe_workers.project_types.arbitrary_geometry import grouping_functions as g
 from mapswipe_workers.project_types.base.project import BaseGroup, BaseProject
 from mapswipe_workers.project_types.base.tile_server import BaseTileServer
```

Comment:
In digitisation projects, the web app will use an (optional) `tileServer.maxZoom` integer to restrict the interactive web map zoom to the maximum level provided by the service.